.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "15:15"
+    timezone: "Europe/Sofia"
+  open-pull-requests-limit: 10
+- package-ecosystem: npm
+  directory: "packages/cryptography"
```

Comment:
You probably want to do an npm up or manually bump up versions and commit that to avoid the flood of initial PRs.

Reply:
Why was this resolved without comment and no code changed to address? It's fine if you prefer to not manually update package.json in this PR and let Dependabot handle it with dozens of PRs as soon as this is merged, but would appreciate a comment if that's your approach.

Reply:
That's because it has been disabled for a while. We are going to combine them into one and try to resolve on one PR. We are going to deal with this into the next days.